app/views/govuk_publishing_components/components/_chart.html.erb

```diff
@@ -75,77 +89,85 @@
     <% end %>
 
     <div class="gem-c-chart__chart" id="<%= chart_id %>">
```

Comment:
If the purpose of the minimal version is just for presentation, and not for conveying information, should it be hidden from the accessibility tree? E.g. give it a `tabindex` so that it can't be focused on, and use `aria-hidden="true"`?

If not, I think the focus style for the minimal version should be improved - for example the yellow is not visible around the whole chart container. Otherwise, I think it will be flagged as an accessibility issue. 

Also, how will users know that this is a link? Will it be next to some content that implies so?

Reply:
Good shout on the first point - have added `aria-hidden` as suggested so now the only thing a screenreader should encounter is the link. Have also improved the focus style for the link to make it look better.

How will users know this is a link? That's an excellent question and I'm not sure of the answer. In the designs we have they're directly underneath some headings, which themselves are also links, so this being a link as well is kind of a backup (although that itself introduces a duplicate link, which isn't great).

I'm left wondering if minimal should be a link at all, but the link is needed to allow users to go to the full graph and table of data - it might as well just be an image if it isn't a link, but we decided to generate a chart otherwise we have to keep updating images every time the chart they link to changes. A bit of a rabbit hole in general.